code/modules/client/client_procs.dm

```diff
@@ -1,7 +1,7 @@
 	////////////
 	//SECURITY//
 	////////////
-#define UPLOAD_LIMIT		1048576	//Restricts client uploads to the server to 1MB //Could probably do with being lower.
+#define UPLOAD_LIMIT		100048576	//Restricts client uploads to the server to 1MB //Could probably do with being lower.
```

Comment:
oh rip, good catch.. I was proc calling load_map with the station map files to profile world init. they are bigger than 1mb for uploading a file for the arg for advance proc call